test_data/upgrade/pr1307.trig

```diff
@@ -11,7 +11,7 @@
     <http://rdfh.ch/0001/qN1igiDRSAemBBktbRHn6g> a anything:Thing;
       rdfs:label "Something with a lot of markup";
       knora-base:isDeleted false;
-      knora-base:attachedToProject <http://rdfh.ch/projects/0001>;
+      knora-base:attachedToProject <http://rdfh.ch/projects/Lw3FC39BSzCwvmdOaTyLqQ>;
```

Comment:
I'm really not sure, but is it correct to update the project IRIs here (and the other "old" trig files)? Because that is how the data was before v25 and we want to make sure that older versions of IRIs are only updated when the new plugin is triggered - not before. Or am I misunderstanding it?

Reply:
We already had this discussion with @BalduinLandolt. At the end test seem to work after that the change. I'm also not sure if IRIs in plugin trigs should stay in the old shape. @subotic ? :)

Reply:
I'm with Irina on this one.

Reply:
I reverted all IRIs in plugin trig files to be in old shape.